app/models/toilet.rb

```diff
@@ -2,5 +2,7 @@ class Toilet < ApplicationRecord
   belongs_to :user
   has_many :bookings
 
+  validates :name, :description, :address, presence: true
```

Comment:
photo should be mandatory too? or not? (validates :photo, presence: true)

Should we require the user too? Even if the "new" action have always something like toilet.user = UserInstance, shouldn't we put the user id with presence true? ( validates :user_id, presence: true)

Reply:
the photo in most of the sites its not mandatory mas we can add that validation. 
I will add that because it's easy to buld the cards and the show page without the logic to handle the cases that there is no photo.
Regarding the toilet_user, it's the same of what i said in the review. I can't see any scenerio that the system can create a toilet without putting that info.

Reply:
you are right about the photo! it's not mandatory! what we can do is to check is toilet.photo is empty and if it's empty, the picture in the show will be one default (and this one can be locally stored, dont need claudinary).
Regarding the toilet-user, we don't need to put them, it will not change anything